source/fundamentals/connection/connect.txt

```diff
@@ -87,11 +87,25 @@ Other Connection Targets
 Connect to Atlas
 ~~~~~~~~~~~~~~~~
 
-In the following example, the driver uses a sample connection URI to
-connect to a MongoDB instance on Atlas, using placeholders for your
-username and password. In this example, the client uses
-the :ref:`Stable API <csharp-stable-api>` feature, though
-you can connect to Atlas without this feature.
+To connect to an Atlas deployment, you need to create a client. You can
+create a client that uses your connection string and other 
+client options by passing a ``MongoClientSettings`` object to the ``MongoClient``
+constructor. To specify your connection URI, pass it to the ``FromConnectionString()``
+method, creating a ``MongoClientSettings`` instance. To specify any other
+settings, set the relevant fields of the settings object.
+
+To learn more about connection options, see the
+:ref:`Connection Options page <csharp-connection-options>`.
+
+You can set the {+stable-api+} version as a setting to avoid
```

Comment:
Suggestion:

I think using the verb "set" to describe an action on a "setting" can be confusing as there are other equivalent words. I think the current wording also attributes the "set" action to "as a setting" which is an incorrect relation and should be "the setting" instead.

Per another comment in this review, it could be helpful to stay consistent in terminology.

```suggestion
You can set the {+stable-api+} version client option to avoid
```